content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -41,27 +41,27 @@ Many applications running for long periods of time eventually transition to
 broken states, and cannot recover except by being restarted. Kubernetes provides
 liveness probes to detect and remedy such situations.
 
-In this exercise, you create a Pod that runs a Container based on the
+In this exercise, you create a Pod that runs a container based on the
 `k8s.gcr.io/busybox` image. Here is the configuration file for the Pod:
 
 {{< codenew file="pods/probe/exec-liveness.yaml" >}}
 
-In the configuration file, you can see that the Pod has a single Container.
+In the configuration file, you can see that the Pod has a single container.
 The `periodSeconds` field specifies that the kubelet should perform a liveness
```

Comment:
@sftim , I read through the changes.
- Mostly, the sentences refer to containers in a general way; I would expect the word container to be in lowercase.
When the content specifically refers to the Container resource, then I would expect capitalization.
For instance, the paragraphs starting at lines 49 and 133. The sentence that starts this paragraph is one place where the wording could be changed and instead mention that the PodSpec defines a `Container` resource (`containers` is an array of Container). The paragraph could explain that the Container has the *probes fields of type Probe (which has the `periodSeconds`, `initialDelaySeconds` fields). Linking to the Container API may help.
- nit: Line 370, In the `What's next` section, what do you think about removing the `Reference` heading or creating a brief sentence explaining the links? I like the links to the resources, but the list seems separate from the link to the `Container Probes` section.

Reply:
Thanks for the feedback!